client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -1514,3 +1519,59 @@ def set_hide_output_label(self, hide):
     def get_hide_output_label(self):
         self._refresh_cache()
         return self._msg.hide_output_label
+
```

Comment:
In [the Jira ticket](https://vertaai.atlassian.net/browse/CAT-155), Andy also requested being able to delete a dataset version (`del_dataset_version()`). Do you know if this is possible through the backend? 🤔

Reply:
Actually, maybe it is possible to do through the client, similar to what we do in `del_artifact()`: https://github.com/VertaAI/modeldb/blob/c8e78ed50153469a171dbfe0091f0725c7a3bd19/client/verta/verta/registry/entities/_modelversion.py#L742-L753